cli/js/lib.deno.ns.d.ts

```diff
@@ -1663,8 +1663,7 @@ declare namespace Deno {
     opsCompletedSync: number;
     opsCompletedAsync: number;
     opsCompletedAsyncUnref: number;
-    bytesSentControl: number;
-    bytesSentData: number;
+    bytesSent: number;
```

Comment:
Because this is part of the stable public interface, I think we need to keep bytesSentControl around... I wonder if it could still be accounted for by just looking at the first zerocopybuf?
We can remove this in 2.0

Reply:
I agree that it should probably be `bytesSentControl = zero_copy[0].len()` and `bytesSentData = <rest>` then.
Adding `bytesSent` does not seem useful enough in that case; should I just leave it out?

Reply:
> Adding bytesSent does not seem useful enough in that case; should I just leave it out?

Yea I think so